packages/notifications/src/NotificationPagination/NotificationPagination.tsx

```diff
@@ -11,7 +10,14 @@ import {
     PaginationVariant
 } from '@patternfly/react-core';
 
-export const NotificationPagination = ({ page, onSetPage, onClearAll, count }) => (
+interface NotificationPaginationProps {
```

Comment:
Same as above, we are not even exporting this interface, so I guess let's port it to `type`